includes/base/atomic-widget-base.php

```diff
@@ -34,6 +34,18 @@ final public function get_initial_config() {
 		return $config;
 	}
 
+	final public function get_raw_data( $with_html_content = false ) {
+		$raw_data = parent::get_raw_data( $with_html_content );
+
+		$data = $this->get_data();
+
+		if ( ! empty( $data['styles'] ) ) {
+			$raw_data['styles'] = $data['styles'];
+		}
+
+		return $raw_data;
```

Comment:
Maybe create a property in the constructor that will save the styles attribute and then us it in this method